File: JavaBasic/88_bmi_category.java
// 88	88_bmi_category.java	Determine BMI category: Underweight / Normal / Overweight / Obese.
public class _88_bmi_category {
    public static void main(String[] args) {
        double weight = 70, height = 1.75;
        double bmi = weight / (height * height);
        System.out.printf("BMI: %.2f%n", bmi);
        if (bmi < 18.5) System.out.println("Category: Underweight");
        else if (bmi < 25) System.out.println("Category: Normal");
        else if (bmi < 30) System.out.println("Category: Overweight");
        else System.out.println("Category: Obese");
    }
}

File: JSBASIC/88_bmi_category.js
// 88	88_bmi_category.js	Determine BMI category: Underweight / Normal / Overweight / Obese.

let weight = 70; // kg
let height = 1.75; // meters
let bmi = weight / (height * height);
console.log("Weight:", weight, "kg | Height:", height, "m");
console.log("BMI:", bmi.toFixed(2));
if (bmi < 18.5) {
    console.log("Category: Underweight");
} else if (bmi < 25) {
    console.log("Category: Normal");
} else if (bmi < 30) {
    console.log("Category: Overweight");
} else {
    console.log("Category: Obese");
}
